docs/INSTALLATION.md

```diff
@@ -0,0 +1,49 @@
+# Installation
+This will walk you through a simple 
+
+## Prerequisites
+
+1. [Node v8+](https://nodejs.org/en/)
+2. [Angular CLI](https://cli.angular.io/)
+
+## Setup Your Angular Project
+
+```bash
+ng new my-store
+```
+
+## Install the @daffodil packages
+
+1. Install Daffodil Modules You Need
+    * `npm install --save @daffodil/product` - Install Product Module
+    * `npm install --save @daffodil/cart` - Install Cart Module
+    * `npm install --save @daffodil/checkout` - Install Checkout Module
+
+## Install Peer Dependencies
+
+1. `npm install --save @ngrx/store @ngrx/effects @ngrx/entity`
+
+## Configure Your App
+1. Add the following to your application's root `AppModule`.
+
+```ts
```

Comment:
Remove ts

Reply:
@xelaint the `ts` marks the codeblock for typescript formatting.